### PR TITLE
a troubleshooting info page

### PR DESCRIPTION
**Pull request by salsakran**
@senior this is the code we paired on but in a working end-to-end state.

The actual clj api endpoint is stubbed out, but in the format the frontend needs.

@kdoh I could use some help making this look a little less like I wrote it.

### CODE REVIEW

File: frontend/src/metabase/admin/TroubleshootingApp.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+class TroubleshootingApp extends Component {
+    componentWillMount () {
+        console.log('Requested state')
+        this.props.getTroubleshootingInfo()
+    }
+    render () {
+        return (
+            <LoadingAndErrorWrapper loading={!this.props.troubleShootingInfo}>
+                { this.props.troubleShootingInfo &&
```
Comment (salsakran) [inline frontend/src/metabase/admin/TroubleshootingApp.jsx]: The fact that this isn't taken care of by the LoadingAndErrorWrapper makes me peeved. If you don't explicitly guard against nulls, you get null pointer errors.

Comment (kdoh) [inline frontend/src/metabase/admin/TroubleshootingApp.jsx]: I think you need to have the child of LoadingAndErrorWrapper be a function. If I understand properly, LAEW calls it's child function when things have "loaded" and that closure prevents the issue you're hitting.
```
{ () => (
 ... your stuff here
}
```

Comment (kdoh) [inline frontend/src/metabase/admin/TroubleshootingApp.jsx]: (It should definitely do a better job of letting you know that's what you need to do though.)

**Comment by camsaul** [comment]
@salsakran @senior are we still thinking about merging this?

**Comment by camsaul** [comment]
@salsakran @senior bump

**Comment by salsakran** [comment]
@kdoh this was a different page. This is for things like timezone information, and any info on the jvm. This should be resurrected and styled

**Comment by camsaul** [comment]
we have troubleshooting info pages now, if we want to add this stuff we can open a new PR because this one is so out-of-date

**Pull request closed without merging**